indico/modules/events/editing/controllers/backend/management.py

```diff
@@ -87,3 +96,43 @@ def _process_DELETE(self):
                 raise ExpectedError(_('Cannot delete the only publishable file type'))
         delete_file_type(self.file_type)
         return '', 204
+
+
+class RHEditingReviewConditions(RHEditingManagementBase):
+    def _process_GET(self):
+        return jsonify(editing_settings.get(self.event, 'review_conditions').items())
+
+    @use_rh_kwargs(EditingReviewConditionArgs)
+    def _process_POST(self, file_types):
+        new_conditions = OrderedDict(editing_settings.get(self.event, 'review_conditions'))
+        new_conditions[unicode(uuid4())] = file_types
+        editing_settings.set(self.event, 'review_conditions', new_conditions)
+        return '', 204
+
+
+class RHEditingEditReviewCondition(RHEditingManagementBase):
+    def _process_args(self):
+        RHEditingManagementBase._process_args(self)
+        uuid = request.view_args['uuid']
+        try:
+            UUID(uuid)
+        except ValueError:
+            raise NotFound
+
+        conditions = editing_settings.get(self.event, 'review_conditions')
+        if uuid not in conditions:
+            raise NotFound
+        self.uuid = uuid
+
+    def _process_DELETE(self):
+        new_conditions = OrderedDict(editing_settings.get(self.event, 'review_conditions'))
+        del new_conditions[self.uuid]
+        editing_settings.set(self.event, 'review_conditions', new_conditions)
+        return '', 204
+
+    @use_rh_kwargs(EditingReviewConditionArgs)
```

Comment:
I'd always use `partial=True` for PATCH endpoints

Reply:
why? This RH always expects `file_types` to be present in the request

Reply:
Indeed, doesn't make much sense then.